support/Pipelines/Bbh/BranchRuns.py

```diff
@@ -0,0 +1,228 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+from pathlib import Path
+from typing import Optional, Sequence, Union
+
+import click
+
+logger = logging.getLogger(__name__)
+
+from spectre.Pipelines.Bbh.Inspiral import (
+    INSPIRAL_INPUT_FILE_TEMPLATE,
+    start_inspiral,
+)
+from spectre.support.Schedule import scheduler_options
+
+# Levs defined in terms of p-refinement
+ALL_LEVS = [
```

Comment:
I think this is a good idea for defining Levs. I suggest you move this definition into `Inspiral.py` and (as a first commit and ideally even as a separate PR) add a `--lev` option to `start-inspiral` so that it can be used as an alternative to the refinement level & polynomial order options.

Note that we probably want to define Levs in terms of AMR tolerances eventually, but we can make this change later.